inc/saiacl.h

```diff
@@ -1419,6 +1419,26 @@ typedef enum _sai_acl_table_attr_t
      */
     SAI_ACL_TABLE_ATTR_AVAILABLE_ACL_COUNTER,
 
+    /**
+     * @brief Start of Table Match Field Mask
+     */
+    SAI_ACL_TABLE_ATTR_FIELD_MASK_START = 0x00003000,
+
+    /**
+     * @brief Dst IPv6 Valid bits
+     *
+     * @type sai_acl_field_data_mask_t sai_ip6_t
+     * @flags CREATE_ONLY
+     * @default ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff
+     * @validonly SAI_ACL_TABLE_ATTR_FIELD_DST_IPV6 == true
+     */
```

Comment:
We should rename this to mean they are valid bits.

Reply:
renamed to valid bits